docs/Community/Governance.md

```diff
@@ -0,0 +1,137 @@
+or the CGC.
+
+If there is conflict, either within or between SIGs, or between any members of
+the community, wherein it is desirable, the issue can be raised to the CGC.
+
+Technical Discussions
+---------------------
+
+*Note(Aeva) section added*
+
+Technical discussions generally happen in one of these places:
+
+* [on the GitHub project](https://github.com/openenclave/openenclave), either in an Issue or a Pull Request
+* in public SIG or WG meetings, which are typically held on Zoom (public calendar TBD)(TODO)
+* on our mailing list or chat server (details TBD) (TODO)
+
+While we strive for openness and transparency, it is natural that some
+discussions will also happen in person (eg, in the hallway track) or other
+private channels. The outcome of such discussions should be considered
+non-binding until the information is shared through an open medium and relevant
+stake-holders have opportunity to contribute to the discussion.
+
+All SIG and WG leaders are responsible for creating and publishing minutes from
+the meetings they hold, ensuring that the meetings are open and inclusive, and
+that decisions they make are fair and transparent.
+
+If a private discussion pertains to an open issue or PR, and results in a
+decision, that item should be updated and reference given to the conversation.
+
+Issue/PR Review Process
+-----------------------
+
+*Note(Aeva) section updated*
+
+The community strives to triage and respond to all incoming issues within one
+week. If you haven't heard anything by then, or an issue or PR is not getting
+any response within a resonable time after it is triaged, please feel free to
+remind us with a ping on the thread.
+
+Security issues should be reported through a separate channel, and
+will receive a response within 24 hours. See [Reporting Security
+Issues](Contributing.md#reporting-security-issues).
+
+Committers and Contributors
+---------------------------
+
+*Note(Aeva) section removed*
+
+At present, we maintain a list of [Project Committers](Committers.md) along with
+descriptions of their knowledge areas.
+
+This will be updated and replaced with a more scaleable approach to
+commit-rights management.
+
+Accepting Contributions
+-----------------------
+
+*Note(Aeva) section updated*
+
+SIGs will generally accept changes that improve the project and that align with the
+[Open Enclave roadmap](https://github.com/openenclave/openenclave/projects).
+
+All contributions are expected to be reviewed by at least one SIG lead other
+than the contributor, and have no significant objections from other SIG leads
+within the same SIG. Objections from leads of other SIGs should be respected,
+and consensus sought (but not required); overruling the objection of another SIG
+lead should only be done after careful consideration.
+
+All contributions must also satisfy the
+[contribution guidelines](Contributing.md), such as having a DCO and being
```

Comment:
undefined acronym.
replace DCO with [Developer Certificate of Origin (DCO)](https://developercertificate.org/)